components/error-summary/src/index.js

```diff
@@ -115,7 +115,7 @@ const StyledErrorSummary = styled('div')(
  */
 const ErrorSummary = ({ onHandleErrorClick, heading, description, errors, ...props }) => (
   <StyledErrorSummary tabIndex={-1} {...props}>
-    <H2>{heading}</H2>
+    <H3>{heading}</H3>
```

Comment:
i've just checked `govuk-frontend` and they actually use an `h2` element in the DOM here.

to better match their markup, this should be changed to `<H3 as="h2">`